lib/msf/ui/console/command_dispatcher/exploit.rb

```diff
@@ -113,6 +113,10 @@ def cmd_exploit(*args)
     mod_with_opts = mod.replicant
     mod_with_opts.datastore.import_options_from_hash(args[:datastore_options])
     rhosts = mod_with_opts.datastore['RHOSTS']
+    has_rhosts_option = mod.options.include?('RHOSTS') ||
```

Comment:
Turns out there's a few ways to specify rhosts:

```
2.7.2 :055 > pp all_modules.flat_map { |mod| mod['options'] }.map { |opt| opt['name'] }.grep(/rho/i).tally
{"RHOST"=>267,
 "RHOSTS"=>2331,
 "rhost"=>6,
 "WARHOST"=>1,
 "PUBLIC-API-RHOST"=>1,
 "RpcServerHost"=>1,
 "rhosts"=>1}
```

The lowercase `rhosts`/`rhosts` options are from the python external modules